pandas/tests/test_strings.py

```diff
@@ -3424,3 +3424,11 @@ def test_method_on_bytes(self):
             expected = Series(np.array(
                 ['ad', 'be', 'cf'], 'S2').astype(object))
             tm.assert_series_equal(result, expected)
+
+    def test_casefold(self):
+        values = Series(['ss', NA, 'case', 'ssd'])
+        s = Series(['ß', NA, 'case', 'ßd'])
+        exp = s.str.casefold()
```

Comment:
`exp` is misleading as a variable name, as this isn't the "expected output".

Something like `result` would be more suitable, I think.

Reply:
thanks, @gfyoung , changed!